collector/supervisord.go

```diff
@@ -98,7 +93,7 @@ func (c *supervisordCollector) isRunning(state int) bool {
 }
 
 func (c *supervisordCollector) Update(ch chan<- prometheus.Metric) error {
-	var infos []struct {
+	var info struct {
 		Name          string `xmlrpc:"name"`
```

Comment:
Do you still need the field annotations now that you are setting these fields manually?

Reply:
We don't need them, but it helps with understanding the code.